### PR TITLE
AMP-101684: Moving banners from the top of our SDK docs

### PR DESCRIPTION
**Pull request by izaaz**
# Amplitude Developer Docs PR

## Description

Moving banners from the top of our sdk docs to advanced topics
- Removed reference to ampli since its available in the left nav
- Moved migration guides to SDK resources
- Moved bundle sizes and min version support to advanced topics

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sdks/android-kotlin/index.md [hunk 1]
```diff
@@ -11,20 +11,8 @@ search:
 The Kotlin Android SDK lets you send events to Amplitude. This library is open-source, check it out on [GitHub](https://github.com/amplitude/Amplitude-Kotlin).
 
 !!!info "Android SDK Resources"
-    [:material-github: GitHub](https://github.com/amplitude/Amplitude-Kotlin) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-Kotlin/releases) · [:material-book: API Reference](https://amplitude.github.io/Amplitude-Kotlin)
+    [:material-github: GitHub](https://github.com/amplitude/Amplitude-Kotlin) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-Kotlin/releases) · [:material-book: API Reference](https://amplitude.github.io/Amplitude-Kotlin) · [Migration guide](/data/sdks/android-kotlin/migration/)
```
Comment (justin-fiedler) [inline docs/data/sdks/android-kotlin/index.md:14]: nit: can we add changelog here too?

File: docs/data/sdks/android-kotlin/index.md [hunk 2]
```diff
@@ -11,20 +11,8 @@ search:
...
 
---8<-- "includes/ampli-vs-amplitude.md"
```
Comment (justin-fiedler) [inline docs/data/sdks/android-kotlin/index.md:16]: The Ampli info block was originally added as Ampli has a different setup process than the standalone SDK. Migration to Ampli can be tedious vs starting with Ampli from the beginning. I would vote to keep this here to inform users upfront about the differences so they can choose the correct place to start.

File: docs/data/sdks/android-kotlin/index.md (hunk 1 above)
Comment (izaaz) [reply docs/data/sdks/android-kotlin/index.md:14]: Change log? Is that not available within releases?

File: docs/data/sdks/android-kotlin/index.md (hunk 2 above)
Comment (izaaz) [reply docs/data/sdks/android-kotlin/index.md:16]: Fair enough. I can keep this here for now until we find a different way to communicate this.

**Review by justin-fiedler — APPROVED** [review]

**Review by Mercy811 — APPROVED** [review]
Thanks Izaaz for fixing all these small errors.

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**